src/datepicker/docs/readme.md

```diff
@@ -100,6 +100,10 @@ The datepicker has 3 modes:
   allowInvalid support. [More on ngModelOptions](https://docs.angularjs.org/api/ng/directive/ngModelOptions).
 
 
+* `ng-model-options`
+  _(Default: {})_ -
+  Timezone support. [More on ngModelOptions](https://docs.angularjs.org/api/ng/directive/ngModelOptions).
```

Comment:
This should be condensed with the `ng-model-options` block directly above, changing the `allowInvalid` and `timezone` mention to sub-bullet points.